src/main/java/mServer/crawler/sender/arte/ArteVideoDetailsDeserializer.java

```diff
@@ -0,0 +1,197 @@
+                    detailsDTO.setBroadcastBegin(elementBegin.getAsString());
+                }
+            }
+        }
+            
+        return detailsDTO;
+    }
+
+    /**
+     * ermittelt Ausstrahlungsdatum aus der Liste der Ausstrahlungen
+     * @param broadcastArray
+     * @return 
+     */
+    private String getBroadcastDate(JsonArray broadcastArray) {
+        String broadcastDate = "";
+        String broadcastBeginFirst = "";
+        String broadcastBeginMajor = "";
+        String broadcastBeginMinor = "";
+
+        // nach Priorität der BroadcastTypen den relevanten Eintrag suchen
+        // FIRST_BROADCAST => MAJOR_REBROADCAST => MINOR_REBROADCAST
+        // dabei die "aktuellste" Ausstrahlung verwenden
+        for(int i = 0; i < broadcastArray.size(); i++) {
+            JsonObject broadcastObject = broadcastArray.get(i).getAsJsonObject();
+
+            if(broadcastObject.has(JSON_ELEMENT_BROADCASTTYPE) && 
+                    broadcastObject.has(JSON_ELEMENT_BROADCAST)) {
+                String value = this.getBroadcastDate(broadcastObject);
+
+                if(!value.isEmpty()) {
+                    String type = broadcastObject.get(JSON_ELEMENT_BROADCASTTYPE).getAsString();
+                    switch(type) {
+                        case BROADCASTTTYPE_FIRST:
+                            broadcastBeginFirst = value;
+                            break;
+                        case BROADCASTTTYPE_MAJOR_RE:
+                            broadcastBeginMajor = value;
+                            break;
+                        case BROADCASTTTYPE_MINOR_RE:
+                            broadcastBeginMinor = value;
+                            break;
+                        default:
+                            LOG.debug("New broadcasttype: " + type);
+                    }
+                }
+            }
+        }
+
+        if(!broadcastBeginFirst.isEmpty()) {
+            broadcastDate = broadcastBeginFirst;
+        } else if(!broadcastBeginMajor.isEmpty()) {
+            broadcastDate = broadcastBeginMajor;
+        } else if(!broadcastBeginMinor.isEmpty()) {
+            broadcastDate = broadcastBeginMinor;
+        }
+
+        // wenn kein Ausstrahlungsdatum vorhanden, dann die erste Ausstrahlung nehmen
+        // egal, wann die CatchupRights liegen, damit ein "sinnvolles" Datum vorhanden ist
+        if(broadcastDate.isEmpty()) {
+            broadcastDate = getFirstBroadcastDateIgnoringCatchupRights(broadcastArray);
+        }        
+        
+        return broadcastDate;
+    }
+    
+    /**
+     * Liefert den Beginn der Ausstrahlung, 
+     * wenn 
+     *  - heute im Zeitraum von CatchUpRights liegt 
+     *  - oder heute vor dem Zeitraum liegt
+     *  - oder CatchUpRights nicht gesetzt ist
+     * @param broadcastObject 
+     * @return der Beginn der Ausstrahlung oder ""
+     */
+    private String getBroadcastDate(JsonObject broadcastObject) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getBroadcastDate" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)